### PR TITLE
Update readme with suggestions from tech writer

### PR DESCRIPTION
**Pull request by quan-nobl9**
Grammatical changes

### CODE REVIEW

**Review by a-hales — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 ## Inputs
 
-### `client_id`
-The Client ID of your nobl9 account
+### `CLIENT_ID`
+The Client ID of your Nobl9 account
 
-### `client_secret`
-The Client Secret of your nobl9 account
+### `CLIENT_SECRET`
```
Comment (ian-bartholomew) [inline README.md]: should be lower cased

File: README.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 ## Requirements
 
-- A valid nobl9 account must be activated (check https://nobl9.com for more information)
+- A valid Nobl9 account (see https://nobl9.com for more information)
 
 ## Inputs
 
-### `client_id`
-The Client ID of your nobl9 account
+### `CLIENT_ID`
```
Comment (ian-bartholomew) [inline README.md]: should be lower cased

File: README.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
 # nobl9-action
 
-This action applies a Nobl9 configuration file (specified at `sloctl_yml` input parameter) to a project using `sloctl` (https://docs.nobl9.com/sloctl-user-guide/) tool.
+This action applies a Nobl9 configuration file (specified via the SLOCTL_YAML input parameter) to a project using the `sloctl` (https://docs.nobl9.com/sloctl-user-guide/) tool.
```
Comment (ian-bartholomew) [inline README.md]: `SLOCTL_YAML` should be lower cased

File: README.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
+The Client ID of your Nobl9 account
 
-### `client_secret`
-The Client Secret of your nobl9 account
+### `CLIENT_SECRET`
+The Client Secret of your Nobl9 account
 
-### `access_token`
-Access token used to authenticate the sloctl tool
+### `ACCESS_TOKEN`
```
Comment (ian-bartholomew) [inline README.md]: should be lower cased

File: README.md [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+The access token used to authenticate to the sloctl tool
 
-### `project`
-The project name on which the sloctl configuration will be applied to
+### `PROJECT`
```
Comment (ian-bartholomew) [inline README.md]: should be lower cased

File: README.md [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
+The name of the project the configuration will be applied to
 
-### `sloctl_yml`
-The path to the sloctl yaml configuration file, relative to the root directory of the repository
+### `SLOCTL_YAML`
```
Comment (ian-bartholomew) [inline README.md]: should be lower cased

**Review by ian-bartholomew — CHANGES_REQUESTED** [review]

**Review by quan-nobl9 — COMMENTED** [review]
reverting back to lowercase

**Review by ian-bartholomew — APPROVED** [review]

**Pull request merged**